### PR TITLE
Undocker part 1)

### PR DESCRIPTION
**Pull request by vrothberg**
Create dedicated packages:
* pkg/signal for signal handling
* pkg/capabilities for all-things caps

Especially `pkg/capabilities` will come in handy for #5206.

### CODE REVIEW

**Comment by vrothberg** [comment]
@rhatdan @giuseppe @mheon @baude PTAL

**Comment by vrothberg** [comment]
The rdoproject.org/github-check is starting to get annoying. Although it's not blocking, it causes the CI to be red for way too long.

**Comment by rhatdan** [comment]
Should this be happening in github.com/containers/common so that buildah can take advantage?

**Comment by vrothberg** [comment]
> Should this be happening in github.com/containers/common so that buildah can take advantage?

I prefer to wait until we separate Podman from libpod. This will allow for Buildah to use more package from libpod.

File: pkg/signal/signal_linux.go [hunk 1]
```diff
@@ -1,8 +1,14 @@
-// +build !mips,!mipsle,!mips64,!mips64le
+// +build linux
```
Comment (haircommander) [inline pkg/signal/signal_linux.go]: should we add in a copyright line at the top, even if it wasn't present in the old file?

Comment (vrothberg) [reply pkg/signal/signal_linux.go]: Absolutely. Thanks for noticing!

Comment (vrothberg) [reply pkg/signal/signal_linux.go]: Done :heavy_check_mark:

**Comment by TomSweeneyRedHat** [comment]
Personally, if the end goal is to get the packages into common eventually, I'd say move 'em now even if other pkg's from Libpod go later.  Your call.

**Comment by rhatdan** [comment]
LGTM
@mheon @TomSweeneyRedHat @giuseppe @haircommander PTAL

**Pull request merged**